src/views/Snv/Exploration/variantColumns.tsx

```diff
@@ -804,10 +804,11 @@ export const renderClinvarToString = (variant: any) => {
   return clinvarStringList.join(',');
 };
 export const renderGeneToString = (variant: any) => {
-  const genes = variant.genes?.hits.edges;
-
-  if (genes?.length && genes[0].node.symbol) {
-    return genes[0].node.symbol;
+  const pickedConsequence = variant.consequences?.hits.edges.find(
+    ({ node }: any) => !!node.picked,
+  )?.node;
+  if (pickedConsequence && pickedConsequence.symbol) {
```

Comment:
Juste une proposition 
```suggestion
  if (pickedConsequence?.symbol) {
```

Reply:
:+1: